colour-codes.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
```

Comment:
Thanks @tellison i'll do that, just learnt about it now, so I see why you had it across your changes :+1: